src/main/kotlin/no/skatteetaten/aurora/gobo/integration/dbh/databaseSchemaResources.kt

```diff
@@ -102,3 +102,10 @@ data class DbhResponse<T>(val status: String, val items: List<T>, val totalCount
     fun isFailure() = status.toLowerCase() == "failed"
     fun isEmpty() = totalCount == 0
 }
+
+data class SchemaDeletionResponse(val id: String, val success: Boolean) {
+    companion object {
+        fun success(id: String) = SchemaDeletionResponse(id, true)
```

Comment:
Jeg ser ikke helt hvorfor man trenger disse når man har data klasser, men det er ikke så nøye for meg egentlig.

Reply:
Kan godt ta det bort, er bare ikke så glad i å sende inn boolean variabler. kan bruke named arguments da, det hjelper.